torch/_utils_internal.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
This needs a comment explaining what this is for. If I understand correctly, the fbcode build is overwriting these stubs with something else, right? If you don't explain why, someone will "helpfully" refactor this file away.

Reply:
this is not fbcode specific (though a lint rule there prompts it). It's part of the official way to do python 2/3 compat http://python-future.org/imports.html

Reply:
oh you meant the whole file of course. yeah i'll add a comment